prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts

```diff
@@ -129,6 +130,12 @@ export class OverviewComponent extends BaseEnrolmentPage implements OnInit {
     this.toastService.openSuccessToast('Your GPID has been copied to clipboard');
   }
 
+  public hasErrors() {
+    const { certificateOrOboSite, deviceProviderOrOboSite } = this.getEnrolmentErrors(this.enrolment);
+    return Object.values({ certificateOrOboSite, deviceProviderOrOboSite })
```

Comment:
I think you'll find the `some` is unnecessary now if you're specifically choosing values:

```
return certificateOrOboSite || deviceProviderOrOboSite;
```

Reply:
Good point, will refactor